packages/authentication/__tests__/authentication.test.js

```diff
@@ -1,11 +1,7 @@
-const addition = require('../src/test');
+'use strict';
```

Comment:
We don't need to declare this. Node.js use stric mode by default and it's a linter error